docker/include/bin/start_faux

```diff
@@ -87,7 +87,23 @@ if [ -n "${options[xdhcp]}" ]; then
     fi
     ip addr show $intf_name
 
-    (while true; do ping -c 1 10.20.0.1; sleep 5; done) &
+    if [ -n "${options[opendns]}" ]; then
+        route add -net 172.17.0.0/16 $intf_name  # default docker ip range.
```

Comment:
I think this should be 10.20.0.0

Reply:
172.17.0.0/16 is the external eth0 ip. For some reason the 10.20.X.X gateway IPs vary too broadly when gateways are restarted for bruteforce

Reply:
I think mine is configured differently (see below).  Should be fine as long as it works in CI and for Karthik... and for the qual lab operators. Can you maybe add a check in that will hard-fail if opendns is specified but the subnet is not 172.17.x.x?  I'm trying to avoid weird submarine bugs out there.  The reason the 10.20.x.x gateways vary so much is they're dynamically allocated and are chosen separately for each new instance of a gateway... so for an extended run they can get arbitrarily large.

```
~/daq$ docker exec daq-gw01 ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
1597: eth0@if1598: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:42:c0:a8:09:04 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 192.168.9.4/24 brd 192.168.9.255 scope global eth0
       valid_lft forever preferred_lft forever
1599: gw01-eth0@if1600: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 6a:11:f8:b3:cb:10 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 10.20.0.71/16 brd 10.20.255.255 scope global gw01-eth0
       valid_lft forever preferred_lft forever
    inet 10.20.92.2/32 scope global gw01-eth0
       valid_lft forever preferred_lft forever
    inet 192.168.84.1/32 scope global gw01-eth0
       valid_lft forever preferred_lft forever
```